src/shared/components/event-management/event-start/event-start.component.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this necessary? Will it be used?

Reply:
Yes, this will be used as we proceed further with task event completion epic.